### ISSUE TITLE
[demonoid] Access to the path is denied. (Config)

### ISSUE THREAD

**Issue by petercontact** [Jackett/Jackett#10845]
**Please use the search bar** at the top of the page and make sure you are not creating an already submitted issue.
Check closed issues as well, because your issue may have already been fixed.

Please read our [Contributing Guidelines](https://github.com/Jackett/Jackett/blob/master/CONTRIBUTING.md) before submitting your issue to ensure a prompt response to your bug.

### Environment

**OS**:

**.Net Runtime**: [.Net-Core/.Net-Framework/Mono]

**.Net Version**:

**Jackett Version**:

**Last Working Jackett Version**:

**Are you using a proxy or VPN?** [yes/no]

### Description

[List steps to reproduce the error and details on what happens and what you expected to happen]

### Logged Error Messages

[Place any relevant error messages you noticed from the logs here.]

[Make sure you attach the full logs with your personal information removed in case we need more information]

### Screenshots

[Place any screenshots of the issue here if needed]

I have an account.
I put https://www.demonoid.is/ on site link
 and
__cfduid=...
on cookie, what am i doing wrong?

**Comment by ilike2burnthing** [comment]
When does this occur? When you try to add the indexer?

Please add the requested information.

**Comment by petercontact** [comment]
Yes, when i try to add the indexer i tried two different browsers, same problem.

**Comment by ilike2burnthing** [comment]
> Please add the requested information.

To be clear your cookie follows the pattern `__cfduid=...; uid=12345; pass=...`? It shouldn't actually matter, as the site is currently open to guests. Try without any cookies.

Can you access the site in browser from the same device and IP address?

Try some of the proxies.
